erlang.spec

```diff
@@ -14,7 +14,7 @@
 
 %global upstream_ver 25.1.1
 %global package_ver  25.1.1
-%global package_ver_release 1
+%global package_ver_release 2
```

Comment:
Please leave this at 1.

Reply:
To be more specific: we can accept a temporary bump and rebuild/republish the existing packages i there are reasons to do so. But then we will go back to to "1" for package version release, which may or may not help your case.

Reply:
Fair enough.